docs/tutorials/get_started_with_qiskit.ipynb

```diff
@@ -0,0 +1,285 @@
+      ],
+      "text/plain": [
+       "<Figure size 700x500 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.visualization import plot_histogram\n",
+    "\n",
+    "plot_histogram(\n",
+    "    job.result().quasi_dists\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77054d4e",
+   "metadata": {},
+   "source": [
+    "And that's it! You now have everything you need to start writing and running quantum programs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5e07a8e",
+   "metadata": {},
+   "source": [
+    "## Connect with the Qiskit community\n",
+    "\n",
+    "Need help or advice? Chat with other Qiskit users and experts on the [Qiskit Slack workspace](https://qisk.it/join-slack)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "3273c661",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<h3>Version Information</h3><table><tr><th>Qiskit Software</th><th>Version</th></tr><tr><td><code>qiskit-terra</code></td><td>0.23.3</td></tr><tr><td><code>qiskit-aer</code></td><td>0.12.0</td></tr><tr><td><code>qiskit-ibmq-provider</code></td><td>0.20.2</td></tr><tr><td><code>qiskit</code></td><td>0.42.1</td></tr><tr><td><code>qiskit-nature</code></td><td>0.6.0</td></tr><tr><td><code>qiskit-finance</code></td><td>0.3.4</td></tr><tr><td><code>qiskit-optimization</code></td><td>0.5.0</td></tr><tr><td><code>qiskit-machine-learning</code></td><td>0.6.1</td></tr><tr><th>System information</th></tr><tr><td>Python version</td><td>3.10.11</td></tr><tr><td>Python compiler</td><td>Clang 14.0.0 (clang-1400.0.29.202)</td></tr><tr><td>Python build</td><td>main, Apr  7 2023 07:31:31</td></tr><tr><td>OS</td><td>Darwin</td></tr><tr><td>CPUs</td><td>8</td></tr><tr><td>Memory (Gb)</td><td>32.0</td></tr><tr><td colspan='2'>Fri Jun 23 14:03:17 2023 BST</td></tr></table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import qiskit.tools.jupyter\n",
+    "%qiskit_version_table"
```

Comment:
In the rendered output, the versions are out of date. I think the notebook needs regenerating. FYI CI will upload an artifact to Azure with the executed notebooks, run in a standard environment.

Reply:
I couldn't find my notebook in the `executed_tutorials` zip, but I've re-run with the latest version of Qiskit.